### DISCUSSION TITLE
Dethrottling Hoymiles hms-1600-4t

### DISCUSSION THREAD

**Discussion by ap20132022** [tbnobody/OpenDTU#2151] [Q&A]
As I found out that the Hoymiles hms-1600-4t inverter hardware similar to HMS 2000, I try to detrhottling it. 
With "Set inverter limit" I can change it to 2000W (125%). But the inverter cut the power of each string at ~ 429W. That means at sum of 1716W. 
Doese anyone have experiences with it? 
I have other inverter from deye and I can dethrottling it. 
Thank you very much for your answer.

**Comment by stefan123t** [comment]
You need to update the inverters internal output limit, the inverter Hardware is mostly identical according to the little known bare circuit shots having been circulated in the past.
We do not know if the manufacturer has a component testing and selection process that he qualifies some models with higher power yield or if this is truly a software limit only to address different market segments. Though the official printed component ratings eg of condesators have been shown to be same across the same model.
You may contact Flole he has created firmware updates for the HM inverters which allow you to run at different output power limits. Know that this will most likely void the 10/12 year warranty!

**Reply by ap20132022** [reply]
> You need to update the inverters internal output limit, the inverter Hardware is mostly identical according to the little known bare circuit shots having been circulated in the past. We do not know if the manufacturer has a component testing and selection process that he qualifies some models with higher power yield or if this is truly a software limit only to address different market segments. Though the official printed component ratings eg of condesators have been shown to be same across the same model. You may contact Flole he has created firmware updates for the HM inverters which allow you to run at different output power limits. Know that this will most likely void the 10/12 year warranty!

thanks for your answer. How can I update the internal output limit?

**Reply by stefan123t** [reply]
> You may contact Flole he has created firmware updates for the HM inverters which allow you to run at different output power limits. Know that this will most likely void the 10/12 year warranty!

Please read my sentence above ^^^

**Reply by ap20132022** [reply]
> > You may contact Flole he has created firmware updates for the HM inverters which allow you to run at different output power limits. Know that this will most likely void the 10/12 year warranty!
> 
> Please read my sentence above ^^^

oh sorry. Now I understand you. 
Who ist Flole?

**Reply by ghost** [reply]
[flole](https://github.com/flole)??

**Reply by stefan123t** [reply]
Dunno if he uses the same handle on github. You can join the Discord, there he comes by every now and then.
